pkg/manager/manager.go

```diff
@@ -36,31 +37,41 @@ type Manager struct {
 	ChangeStore             *store.ChangeStore
 	DeviceStore             *topocache.DeviceStore
 	NetworkStore            *store.NetworkStore
+	ModelRegistryObj        *modelregistry.ModelRegistry
```

Comment:
I think you can't make it a child of **manager** - it has to be a peer of manager, started from Main, like NewModelRegistry or something and accessible as a global variable